pkg/webhook/validate_test.go

```diff
@@ -680,8 +671,7 @@ func TestValidatePolicyForNodeStateWithValidVFAndNetFilter(t *testing.T) {
 		},
 	}
 	g := NewGomegaWithT(t)
-	ok, err := validatePolicyForNodeState(policy, state, NewNode())
+	err := validatePolicyForNodeState(policy, state, NewNode())
 	g.Expect(err).NotTo(HaveOccurred())
-	g.Expect(ok).To(Equal(true))
 	g.Expect(interfaceSelected).To(Equal(true))
 }
```

Comment:
Should we add more tests for `validateNicModel`, since now it returns an error instead of a boolean?

Reply:
what sort of tests are you looking to add? Imo, these functions test validateNicModel quite heavily.

Reply:
`validatePolicyForNodeState` gets tested. But in `err := validateNicModel(&policy.Spec.NicSelector, &iface, node)`, the `err` gets thrown away. I think it would be worth it to test `validateNicModel` indepedently.

Reply:
I agree that we should test that out but maybe not in this PR. In my other PR I  test this when the variable interfaceSelected is removed. Please see this line here: https://github.com/k8snetworkplumbingwg/sriov-network-operator/pull/434/files#diff-ae978883f1787164fb6fa85b31c5fde6421f8a06d9c9f5f013fb72e6f21863a0R560

I think we can leave the tests as is for now, and then we can add this test in the new PR.

Reply:
Ok